src/components/ToolMenu/Draw/index.tsx

```diff
@@ -110,8 +110,9 @@ export const Draw: React.FC<DrawProps> = ({
     }
   };
 
-  const onUploadChange = (e: ChangeEvent<HTMLInputElement>) => {
+  const checkValidityOfUploadFile = (e: ChangeEvent<HTMLInputElement>) => {
     const uploadedFiles = e.target.files;
+    let validitiy = false;
```

Comment:
Is it maybe better to make use of a state here?

Reply:
Not sure on how it would be implemented. This keeps the function simple and readable.

Reply:
No this method is used in a condition and should not have any side effects.